### DISCUSSION TITLE
[04. 아키텍처] - 서비스 데이터베이스에서의 fk 사용

### DISCUSSION THREAD

**Discussion by 5uhwann** [Learning-Is-Vital-In-Development/23-6-RealMySQL#14] [General]
4.2.2 외래 키 지원
외래 키는 데이터베이스 서버 운영의 불편함 때문에 서비스용 데이터베이스에서는 생성하지 않는 경우도 자주 있다고 합니다.
어떤 불편함이 있고 실제로 서비스 데이터베이스에서 fk를 사용하지 않는 경우가 많은지 궁금합니다

**Comment by dojinyou** [comment]
#12
